SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialConnectionManagerTest.cpp

```diff
@@ -0,0 +1,293 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+#include "Interop/Connection/SpatialConnectionManager.h"
+#include "CoreMinimal.h"
+
+#define CONNECTIONMANAGER_TEST(TestName) \
+	GDK_TEST(Core, SpatialConnectionManager, TestName)
+
+class FTemporaryCommandLine
```

Comment:
Wouldn't add the capital F here. I the usually denotes the class is  USTRUCT.

Reply:
Interesting. We decided to use the Unreal code standard for naming things in Stavka (although some things in the standard are pretty weird). And it says `Most other classes are prefixed by F, though some subsystems use other letters.` We've also had some discussions about where exactly we should use that and ultimately decided to go with this for most classes.
I suppose you're interpreting the standard differently then?

Reply:
Ahh interesting haha. I think we generally do not prefix our classes with "F" in the GDK. However, my understanding of unreal prefixes still does not seem entirely correct. It does seem like the GDK could have prepended "F" to it's classes and kept in line with Unreal standards.

Reply:
I have just found cases in the GDK that does and doesn't do this so I think there is no issue.